togetherly/lib/models/person.dart

```diff
@@ -16,12 +16,14 @@ enum ProfileIcon {
 abstract class Person {
   final int? id;
   final int familyId;
+  final int pin;
```

Comment:
This does not match the type of the corresponding column in the database. Should the pin be stored as a string (as it is in the database) or an int?

Reply:
Yes, this should be a string. I'm only commenting on this because I need this code to finish my work on authentication, so I was just looking through this PR.